ui/lib/core/addon/templates/components/icon.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="hs-icon {{class}}" ...attributes>
```

Comment:
what does `hs` stand for? hashicorp structure? 😛

Reply:
Yep!

Reply:
`vlt-` is a little more clear to me, but 🤷‍♀